Automattic/wp-calypso
# Odyssey Stats: Gracefully degrade subscriber highlights section

Related to #78211.

## Proposed Changes

<img width="1266" alt="image" src="https://github.com/Automattic/wp-calypso/assets/4044428/d35d5db0-c021-45fc-b815-d4c7b206cdda">

* Improved loading and error states for the subscriber highlights component in Calypso and Odyssey.

## Testing Instructions



* Ensure that the subscriber section of Calypso Stats works as expected.
* Ensure that the subscriber section of Odyssey Stats degrades gracefully. 
    * Build the branch for Odyssey
        * `STATS_PACKAGE_PATH=~/path/to/jetpack/projects/packages/stats-admin yarn dev`
    * Build Jetpack if necessary
        * `jetpack build plugins/jetpack`
    * Open `/wp-admin/admin.php?page=stats`.

## Pre-merge Checklist
